hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/BulkInsertDataSetWriter.java

```diff
@@ -248,8 +249,18 @@ public TxnView getTxn(){
                 assert rowAndIndexes instanceof SparkDataSet;
 
                 partitionUsingRDDSortUsingDataFrame(bulkImportPartitions, rowAndIndexes, hfileGenerationFunction);
+                Map<Long, Boolean> granted = new HashMap<>();
+                try {
+                    if (token != null && token.length > 0) {
+                        granted = grantCreatePrivilege(allCongloms);
```

Comment:
I think we should grant the Create privileges unconditionally in H10PartitionAdmin.grantPrivilegesIfNeeded() and then add checks in all places where the CREATE privilege is used: preAddColumn, preBulkLoad, preCreateTable, preDropTable...

The bulkLoad* methods we would allow if the user has the right token, all the rest we would disallow unless it's the admin user.

Reply:
When namespace privilege was grated at H10PartitionAdmin.grantPrivilegesIfNeeded() during startup, we don't know which tables need CREATE privilege. We don't want to grant CREATE privileges for ALL tables to the user that run bulk load.

Reply:
That makes sense, thanks!